org.blackfennec.extensions.core.yml

```diff
@@ -0,0 +1,18 @@
+id: org.blackfennec.extensions.core
+branch: '0.10'
+runtime: org.blackfennec.app
+runtime-version: 'stable'
+sdk: org.gnome.Sdk//43
+build-extension: true
+modules:
+  - name: blackfennec.extensions.core
+    buildsystem: simple
+    build-commands:
+      # will be enabled with version 0.11
+      #- pip install --prefix=${FLATPAK_DEST} .
+      - install -Dm644 ./org.blackfennec.extensions.core.metainfo.xml ${FLATPAK_DEST}/share/metainfo/org.blackfennec.extensions.core.metainfo.xml
```

Comment:
so right now this package just install the appstream file in `/app/share/metainfo`.

It doesn't look right.

Reply:
we will uncomment line 12 in approx. 3 weeks for release 0.11. At the same time we will also remove line 29 and 30 of [org.balckfennec.app](https://github.com/flathub/org.blackfennec.app/blob/master/org.blackfennec.app.yml). The two lines are in conflict with each other.